components/Pages/EditoHeader.vue

```diff
@@ -13,30 +13,42 @@
             class="bg-[#0D0C4F]/20 hover:bg-[#0D0C4F]/30 fr-raw-link text-white font-medium text-lg px-5 py-3 rounded-4xl flex justify-start items-start sm:items-center gap-2"
             :href="linkUrl"
           >
-            <RiInformation2Line class="size-6" />
+            <RiInformation2Line
+              class="size-6"
+              aria-hidden="true"
```

Comment:
Same here… If we do this here, why not do it on all the icons?

Reply:
We should, all icons or images should have a label (or alt with some content for img tag) if they are used to provide information or be aria-hidden (or an empty alt) if they are only added for illustration only